include/core/parameters.h

```diff
@@ -624,6 +624,13 @@ namespace Parameters
     // A boolean parameter that enables the calculations of laser heat source
     bool activate_laser;
 
+    // Type of laser model
+    enum class LaserType
+    {
+      exponential_decay,
+      material_interface_vof
```

Comment:
Can you add a quick comment next to the enum or in the description of the enum

Reply:
Like explaining further that ``exponential_decay`` is a volumetric contribution and ``material_interface_vof`` is a surface one?